app/services/avatar_generator.rb

```diff
@@ -0,0 +1,12 @@
+class AvatarGenerator
+  URL = 'https://api.adorable.io/avatars/80/'
+  FILE_FORMAT = '.png'
```

Comment:
1. Format nie powinien zawierać kropki.
2. Stronka api sugeruje, że można użyć url-a bez rozwinięcia nazwy pliku. Może `FILE_FORMAT` w ogóle nie jest potrzebny?
3. Jeżeli użyjesz `IMAGE_SIZE` (patrz wyżej) to tu zmień proszę na `IMAGE_FORMAT`, żeby było konsekwentnie.